# Uart number of stop bits types
# uart_number_of_stop_bits_t - enumeration
uint8 value

#Pre-Defined ENUM Constants
	uint8 uart_stop_bits_Current    = 0 #  use the current number of stop bits on selected port
	uint8 uart_stop_bits_NotUsed    = 1
	uint8 uart_stop_bits_One        = 2
	uint8 uart_stop_bits_Two        = 3
	uint8 uart_stop_bits_OneAndHalf = 4
